paging/src/commonMain/kotlin/com/plusmobileapps/paging/CachedPageLoader.kt

```diff
@@ -14,21 +13,21 @@ import kotlin.time.Duration
  */
 interface CachedPageLoader<INPUT, DATA> {
 
-    /** A [StateFlow] for the current state of the [CachedPageLoader]. */
-    val state: StateFlow<PagingDataSourceState<DATA>>
+    /** A [Flow] for the current state of the [CachedPageLoader]. */
+    val state: Flow<PagingDataSourceState<DATA>>
 
     /**
      * Will clear all results and load the first page with the provided input.
      *
      * @param input The input to be used in each [PageLoaderRequest.input].
      */
-    fun clearAndLoadFirstPage(input: INPUT)
+    suspend fun clearAndLoadFirstPage(input: INPUT)
```

Comment:
The cached page loader doesn't always clear when this is called, consider renaming to initialize?